modules/installing-ocp-agent-ibm-z.adoc

```diff
@@ -18,4 +18,82 @@ Depending on your {ibm-z-name} environment, you can choose from the following op
 [NOTE]
 ====
 Currently, ISO boot support on {ibm-z-name} (`s390x``) is available only for {op-system-base-full} KVM, which provides the flexibility to choose either PXE or ISO-based installation. For installations with z/VM and Logical Partition (LPAR), only PXE boot is supported.
+====
+
+[id="networking-reqs-ibmz_{context}"]
+==  Networking requirements for {ibm-z-title}
+
+In {ibm-z} environments, advanced networking technologies like Original Storage Architecture (OSA), HiperSockets, and Remote Direct Memory Access (RDMA) over Converged Ethernet (RoCE) require specific configurations that deviate from the standard settings used in the Agent Based Installer deployments. These overrides are necessary to accommodate their unique requirements and ensure a successful and efficient deployment on {ibm-z}.
```

Comment:
It seems that just {ibm-z} does not resolve to anything. Looking at our [attributes file](https://github.com/openshift/openshift-docs/blob/main/_attributes/common-attributes.adoc#L269-L270), you need to choose either {ibm-z-name} for the product title with the copyright icon or {ibm-z-title} for the title without it:
```suggestion
In {ibm-z-title} environments, advanced networking technologies like Original Storage Architecture (OSA), HiperSockets, and Remote Direct Memory Access (RDMA) over Converged Ethernet (RoCE) require specific configurations that deviate from the standard settings used in the Agent Based Installer deployments. These overrides are necessary to accommodate their unique requirements and ensure a successful and efficient deployment on {ibm-z-title}.
```